Web/Edubase.Web.UI/Areas/Establishments/Views/EstablishmentsSearch/Downloads/SelectCustomFields.cshtml

```diff
@@ -54,32 +65,33 @@
             <h2 class="govuk-heading-m">Select the data you are interested in</h2>
         </div>
     </div>
-    @using (Html.BeginForm("PrepareDownload", "Search", FormMethod.Get, new Dictionary<string, object> { ["@submit"] = "handleSubmit", ["id"] = "field-selection" }))
+    @using (Html.BeginForm("PrepareDownload", "EstablishmentsSearch", FormMethod.Get,
```

Comment:
Renaming `Search` -> `EstablishmentSearch` appears to be the only non-whitespace change in this file/commit.

There are similar changes renaming `Search` -> `GovernorSearch` / `GroupSearch`.

This parameter appears to be `controllerName`.

@ScottDawsonDfE -- Will changing these controller names have an impact elsewhere? 

I don't see a corresponding rename of files/classes or similar, but I am unfamiliar with this part of the project so not 100% sure what I'm looking for!

Reply:
There was some weirdness going on before, the methods didn't exist in the controller being pointed to. Thanks to the checking carried out in Rider I was alerted to the wierdness. This is a fix to this.
The changes point the form to point to the correct method in the correct controller.
It is not clear to me how these technically work, but the changes have been manually checked and they seem to be working fine.